### DISCUSSION TITLE
Release the HUD

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#537] [Ideas]
I realise that the HUD is currently a bit buggy with clouds and everything, just like the zibo, but is coded and ready to go. But it would be great if it was implemented, most people cope with the zibo’s HUD just fine, so I don’t think the issues with h Mag787’s HUD will affect people that much. It would be very cool to see a HUD in the plane, even if it’s not used a huge amount. Looking forward to this, so are lots of people! Thanks.

**Comment by aeroniemi** [comment]
Making a HUD is quite a large amount of work, and the technology used to do it right now may very soon be out of date - XP12/libhud may make it feasable in the future though. No development time has as-yet been spent on a HUD, so it would be an entirely new project.

**Reply by Simleybob** [reply]
Oh okay, sorry, SpeedM! Said that it was basically done, but just buggy with the clouds and everything so u r not going to include it. Looking forward to this, thanks.
